src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -46,9 +46,9 @@
         </service>
 
         <service id="routing.loader" class="Symfony\Bundle\FrameworkBundle\Routing\DelegatingLoader" public="true">
-            <argument type="service" id="controller_name_converter" />
             <argument type="service" id="routing.resolver" />
             <argument type="collection" />
+            <argument type="service" id="legacy_controller_name_converter" /> <!-- deprecated since Symfony 4.4 -->
```

Comment:
let's add a leading dot to hide the service

Reply:
done